misc/teleop_twist_keyboard/launch/keyboard_teleop.launch

```diff
@@ -1,9 +1,10 @@
 <launch>
     <arg name="robot" default="caffeine"/>
 
+    <include file="$(find twist_mux)/launch/twist_mux.launch" ns="caffeine"/>
+
```

Comment:
Since we call the `twist_mux` when we start gazebo, there is no need to call it again when using `keyboard_teleop`. Can you remove this include?